Change to README.md to create new branch

pulpopus opened the pull request:
Changed the README file slightly to add information, & in order to
create two branches to make a pull request.

[] review README to determine if it is missing any instructions for how to run flask app

pulpopus committed:
Change to README.md to create new branch

Changed the README file slightly to add information, & in order to
create two branches to make a pull request.

Muaad2001 commented:
let reader know to switch python interpreter if flask run doesn't work after following steps.

pulpopus committed:
Addressed review comment

Addressed review comment and added instruction for errors in running the
Flask app
